desktop/libs/librdbms/java/bin/dbproxy

```diff
@@ -20,7 +20,11 @@ set -e
 
 export DBPROXY_HOME=$(cd $(dirname $0)/../.. && pwd)
 
-export CLASSPATH=$DBPROXY_HOME/java-lib/dbproxy-1.0.jar:$CLASSPATH
```

Comment:
Avoid the duplication?

e.g.

export CLASSPATH=$DBPROXY_HOME/java-lib/dbproxy-1.0.jar:$CLASSPATH

if [ -n "$1" ]; then
  export CLASSPATH="$1:$CLASSPATH"
fi

?